setup.py

```diff
@@ -164,65 +174,65 @@ def main(**extra_args):
-                          'dipy.data',
-                          'dipy.utils',
-                          'dipy.data.tests',
-                          'dipy.utils.tests',
-                          'dipy.fixes',
-                          'dipy.external',
-                          'dipy.external.tests',
-                          'dipy.segment',
-                          'dipy.segment.benchmarks',
-                          'dipy.segment.tests',
-                          'dipy.sims',
-                          'dipy.sims.tests',
-                          'dipy.denoise',
-                          'dipy.denoise.tests',
-                          'dipy.workflows',
-                          'dipy.workflows.tests'],
-
-          ext_modules = EXTS,
+          packages=['dipy',
+                    'dipy.tests',
+                    'dipy.align',
+                    'dipy.align.tests',
+                    'dipy.core',
+                    'dipy.core.tests',
+                    'dipy.direction',
+                    'dipy.direction.tests',
+                    'dipy.tracking',
+                    'dipy.tracking.local',
+                    'dipy.tracking.local.tests',
+                    'dipy.tracking.tests',
+                    'dipy.tracking.benchmarks',
+                    'dipy.reconst',
+                    'dipy.reconst.benchmarks',
+                    'dipy.reconst.tests',
+                    'dipy.io',
+                    'dipy.io.tests',
+                    'dipy.viz',
+                    'dipy.viz.tests',
+                    'dipy.testing',
+                    'dipy.testing.tests',
+                    'dipy.boots',
+                    'dipy.data',
+                    'dipy.utils',
+                    'dipy.data.tests',
+                    'dipy.utils.tests',
+                    'dipy.fixes',
+                    'dipy.external',
+                    'dipy.external.tests',
+                    'dipy.segment',
+                    'dipy.segment.benchmarks',
+                    'dipy.segment.tests',
+                    'dipy.sims',
+                    'dipy.sims.tests',
+                    'dipy.denoise',
+                    'dipy.denoise.tests',
+                    'dipy.workflows',
+                    'dipy.workflows.tests'],
+
+          ext_modules=EXTS,
           # The package_data spec has no effect for me (on python 2.6) -- even
           # changing to data_files doesn't get this stuff included in the source
           # distribution -- not sure if it has something to do with the magic
           # above, but distutils is surely the worst piece of code in all of
           # python -- duplicating things into MANIFEST.in but this is admittedly
           # only a workaround to get things started -- not a solution
-          package_data = {'dipy':
-                          [pjoin('data', 'files', '*')
-                          ]},
+          package_data={'dipy':
```

Comment:
As you can see [here](https://www.python.org/dev/peps/pep-0008/#other-recommendations), PEP 8 recommend not having spaces around = in a keyword argument or a default parameter value. I correct it everywhere for this reason. Are you sure it needs a revert ?